api-core/src/main/java/com/vimeo/networking2/config/RetrofitSetupModule.kt

```diff
@@ -82,6 +83,32 @@ object RetrofitSetupModule {
         vimeoApiConfiguration.cache?.evictAll()
     }
 
+    /**
+     * Clear the request cache entry associated with the [VimeoApiConfiguration] and the provided [uri].
+     *
+     * For example, if the [VimeoApiConfiguration.baseUrl] is set to `https://api.vimeo.com`, and the [uri]
+     * provided is `/users/12345`, then the cache will be cleared for `https://api.vimeo.com/users/12345`.
```

Comment:
I realize that it isn't a pattern that is currently followed in this file, but shouldn't we have `@param` documentation here?